iset.mm

```diff
@@ -130628,16 +130628,16 @@ Norman Megill (2007) section 1.1.3.  Megill then states, "A number of
 htmldef "DECID_in" as " <SMALL><U>DECID</U><SUB>in</SUB></SMALL> ";
   althtmldef "DECID_in" as " <SMALL><U>DECID</U><SUB>in</SUB></SMALL> ";
   latexdef "DECID_in" as
-           " \normalfont\textsc{\underline{decid}$_\mathrm{in}$}} ";
+           " \normalfont{\textsc{\underline{decid}$_\mathrm{in}$}} ";
 htmldef "Delta0" as "&Delta;<sub>0</sub>";
   althtmldef "Delta0" as "&Delta;<sub>0</sub>";
   latexdef "Delta0" as "\Delta_0 ";
 htmldef "Bdd" as "<SMALL>BOUNDED</SMALL> ";
   althtmldef "Bdd" as "<SMALL>BOUNDED</SMALL> ";
-  latexdef "Bdd" as "\normalfont\textsc{bounded}} ";
+  latexdef "Bdd" as "\normalfont{\textsc{bounded}}";
```

Comment:
not necessary but ok (as for the next one)

Reply:
you mean the curly brackets or <code>\\normalfont</code>?

Reply:
No, actually you're right: your change is good and necessary.

Things like `$\frac1{\sqrt2}\mathrm a b$` work as expected, but it's unnecessary risky. (The way LaTeX works is that it knows that \frac takes two arguments, so it takes the next two tokens available.) Sometimes, I use things like `\frac12`, but that's it. When in doubt, put braces anyway.